devicetypes/gs/gatorsystem-homewatcher.src/gatorsystem-homewatcher.groovy

```diff
@@ -0,0 +1,109 @@
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+metadata {
+	definition (name: "GatorSystem HomeWatcher", namespace: "GS", author: "GS_coder000") {
+	capability "Motion Sensor"
+	capability "Battery"
+	capability "Contact Sensor"
+	capability "Presence Sensor"   
+	// Raw Description
+        fingerprint deviceJoinName: "GatorSystem Multipurpose Sensor", manufacturer: "GatorSystem", model: "GSHW01"
+	}
+}
+def parse(String description) {
+	log.debug "${device.displayName} description: $description"
+	Map map = [:]
+	if (description?.startsWith('catchall:')) { //raw commands that smartthings does not or cannot interpret
+	  map = zigbee.parseDescriptionAsMap(description)   
+	}
+	else if (description?.startsWith('read attr -')) {
+	  map = zigbee.parseDescriptionAsMap(description)
+	}
+	else if (description?.startsWith('zone status')) {
+	  map = parseIasMessage(description)
+	}
+	log.debug "Parse returned map $map"
+	if (map != null) {    
+		def result = map ? createEvent(map) : null
+        	return result
+	}
+}
+private Map parseIasMessage(String description) {
+	ZoneStatus zs = zigbee.parseZoneStatus(description)
+	Map resultMap = [:]
+	def seconds01 = 60
+    	def seconds02 = 120
+	def resultOne = '0x8000'
```

Comment:
Could You rename all those resultOne/Two/Three/Four/Five variables so they would be a bit more descriptive ?